fw/preamp/src/main.c

```diff
@@ -134,6 +127,30 @@ void init_i2c(int preamp_addr)
 	//I2C_InitStructure1.I2C_Timing = 0x10805E89; // From 8-Mhz HSI clock with 100ns rise, 10ns fall -> 100 KHz
```

Comment:
Where did this number come from and why is it set for only one of the two I2C's?